kubernetes/coredns.yaml.sed

```diff
@@ -111,7 +111,7 @@ spec:
             topologyKey: kubernetes.io/hostname
       containers:
       - name: coredns
-        image: coredns/coredns:1.6.7
+        image: coredns/coredns:1.6.9
```

Comment:
Bumping the CoreDNS version in Kubernetes is part of another process that involves additional testing.

Reply:
Okay, removed the change.